xc-bin/src/jailfile/directives/run.rs

```diff
@@ -119,7 +118,7 @@ impl Directive for RunDirective {
         let (stderr_a, stderr_b) = pipe()?;
         let (stdin_a, stdin_b) = pipe()?;
 
-        let kq = unsafe { nix::libc::kqueue() };
+        let kq = unsafe { freebsd::nix::libc::kqueue() };
```

Comment:
Is there any particular reason to continue to use raw kqueue file descriptors?

Reply:
there are places where using raw file descriptors helps dealing with the borrow checker. If this particular case I am just trying to no risk breaking existing code